### ISSUE TITLE
Make it a library

### ISSUE THREAD

**Issue by paralin** [felixge/go-xxd#10]
Would be very useful to be able to import & use this as a package!

**Comment by felixge** [comment]
I agree! If you send a patch, I'd be happy to hit the merge button with few questions asked :).
